.applier/files/image-pull-secret.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
Should we use [this openshift-template](https://github.com/redhat-cop/openshift-templates/blob/master/secrets/secret-docker-cfg.yml) instead of a custom file?